tests/README.md

```diff
@@ -0,0 +1,32 @@
+To run the test suite:
+
+    py.test -x -vvv --doctest-modules --pyargs adlfs tests
+
+This test suite uses [VCR.py](https://github.com/kevin1024/vcrpy) to record the
+responses from Azure. Borrowing from VCR's
+[usage](https://vcrpy.readthedocs.io/en/latest/usage.html#record-modes), this
+test suite has four recording modes: `once`, `new_episodes`, `none`, `all`. The
+recording mode can be changed using the `RECORD_MODE` environment variable when
+invoking the test suite (defaults to `none`).
+
+To record responses for a new test without updating previous recordings:
+
+    RECORD_MODE=once py.test -x -vvv --doctest-modules --pyargs adlfs tests
+
+To record responses for all tests even if previous recordings exist:
+
+    RECORD_MODE=all py.test -x -vvv --doctest-modules --pyargs adlfs tests
+
+When recording new responses, you will need valid Azure credentials. The following
+environment variables should be defined:
+
+* `azure_username`
+* `azure_password`
+* `azure_store_name`
```

Comment:
Minor: this should be updated to reflect that it is an ADLS (Azure Data Lake Store) account name, since there could be confusion with the actual "Azure Store" which is not the same thing.